cmd/2019_7/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [the file doesn't have an appropriate header](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)